src/GuiBase/BaseApplication.cpp

```diff
@@ -25,6 +25,7 @@
 #include <GuiBase/Utils/KeyMappingManager.hpp>
 #include <PluginBase/RadiumPluginInterface.hpp>
 
+#include <IO/CameraLoader/CameraLoader.hpp>
```

Comment:
Why the cameraLoader is part of IO ? Justify why this must be in the Base Radium and not given as an external plugin.